prow/jobs/kyma/kyma-integration-gardener-gcp.yaml

```diff
@@ -0,0 +1,40 @@
+# Kubectl 1.16
+base_image: &base_image
+  image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20200206-22eb97a4
+
+periodics:
+- name: kyma-gardener-gcp-integration
```

Comment:
I think the naming should be consistent everywhere. File name is `kyma-integration-gardener-gcp` and job name is `kyma-gardener-gcp-integration`

Reply:
Fixed thanks!